build-tools/debian-metadata/oss-xamarin.android-targets.install

```diff
@@ -0,0 +1,11 @@
+bin/Release/bin/*					/usr/bin
```

Comment:
> If we're not shipping host-Linux... then I don't think *any* of the files I noticed in the oss-xamarin.android bundle were platform specific

We're not shipping `host-Linux` *yet*, but *we will* (eventually; C10 timeframe?), so this needs consideration. `cross-arm`/`llc`/etc., on the other hand, *are* platform-dependent files, and thus shouldn't go into `$prefix/share`.

Or I'm not sure what you mean by "binaries that would need to be rebuilt on amd64, i386, etc."

>  I don't see a path to getting XA into distribution repositories (due to the hard dependency on an Internet connection during the build)

This should be solvable. Just not *yet*.

> The norm would be /usr/lib/packagename

What would be "packagename"? `oss-xamarin.android`?

I think what we should consider doing is "re-thinking" which files are "bin" files, such that all but `xabuild` are placed into `bin/$(Configuration)/lib/mandroid`, e.g. `bin/Debug/lib/mandroid/cross-arm`, and updating our build system accordingly. This would "skip" `$prefix/share` as a construct, but that also means that our install structure is closer-in-spirit to what Windows does as well...